.github/workflows/test.yml

```diff
@@ -0,0 +1,104 @@
+name: test
+
+on: [pull_request]
+
+jobs:
+
+  test:
+
+    name: PostgreSQL ${{ matrix.pg }}, PostGIS ${{ matrix.pgis }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            pg: [ '9.6', '10', '11' ]
+            pgis: [ '2.3', '2.4', '2.5' ]
+            perl: [ '5.24', '5.26' ]
+            os: [ ubuntu-18.04 ]
+    env:
+      # Set PATH as postgresql-server-dev-all pretends version is 11
+      PATH: /usr/lib/postgresql/${{ matrix.pg }}/bin:/bin:/usr/bin:/usr/local/bin
+      APT: "apt-get -o Dpkg::Progress=0 -o Dpkg::Use-Pty=0"
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependencies
+      run: |
+        sudo ${APT} -qq purge \
+            libpq* \
+            *php* \
+            postgresql*
+        sudo rm -rf /var/lib/postgresql
+        sudo ${APT} -qq install curl ca-certificates gnupg
+        distro=`lsb_release -cs`
+        # Enable fetching packages from apt.postgresql repository
+        curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+        echo "deb http://apt.postgresql.org/pub/repos/apt/ ${distro}-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+        # Enable fetching packages from packagecloud test repository
+        curl -Ls https://packagecloud.io/linz/test/gpgkey | sudo apt-key add -
+        echo "deb https://packagecloud.io/linz/test/ubuntu ${distro} main" | sudo tee /etc/apt/sources.list.d/linz-test.list
+        # Enable fetching packages from packagecloud prod repository
+        curl -Ls https://packagecloud.io/linz/prod/gpgkey | sudo apt-key add -
+        echo "deb https://packagecloud.io/linz/prod/ubuntu ${distro} main" | sudo tee /etc/apt/sources.list.d/linz-prod.list
```

Comment:
Why are we installing both the prod and test package repos here?

Reply:
It may be good to test both production and testing packages, but I guess we could do it in different matrix configurations or we'd end up only testing the test packages.

Anyway,the build currently fail because `E: Package 'postgresql-9.6-postgis-2.3-scripts' has no installation candidate` which is weird, as it works in Travis ... uhm

Reply:
With commit a53f4e6f768dfe8b4b51bdf4e9e705792b191752 I used a matrix for test/prod packages

Reply:
Ok, next failure: Unable to locate package postgresql-10-dbpatch -- this is because we have that package for `bionic` only in the `test` repo, which means in order to have a success here we should either:

   1. Promote package to production
or
   2. Use both `prod` and `test` repos (which was done before I switched to the matrix approach).

@imincik and @palmerj which one do you prefer ?

Reply:
I prefer that we are testing against both the test and prd packages.